src/classes/User.js

```diff
@@ -0,0 +1,29 @@
+class User {
```

Comment:
Nicely done building out this class. All properties/methods are implemented according to the rubric. Great work!